src/video/windows/SDL_windowskeyboard.c

```diff
@@ -863,15 +863,15 @@ IME_SendEditingEvent(SDL_VideoData *videodata)
         size_t len = SDL_min(SDL_wcslen(videodata->ime_composition), (size_t)videodata->ime_cursor);
 
         size += sizeof(videodata->ime_readingstring);
```

Comment:
Shouldn't `size` be increased by sizeof(WCHAR), instead of adding it to the mallocs?

Reply:
That was an option, but I decided to avoid doing that because `size` is also used in `SDL_wcslcat`, and I wasn't able to test that part of the branch.
So I went the safe route of just increasing the allocated size.